crates/router/src/connector/stripe/transformers.rs

```diff
@@ -133,6 +140,8 @@ pub struct PaymentIntentRequest {
     pub off_session: Option<bool>,
     #[serde(rename = "payment_method_types[0]")]
     pub payment_method_types: Option<StripePaymentMethodType>,
+    #[serde(rename = "expand[0]")]
+    pub expand: Option<ExpandableObjects>,
```

Comment:
We are using setup intent object as well incase of setup mandate flow, should this be covered there also? Similarly we have SetupIntentResponse object also In PSync flow

Reply:
Yes I have added it in setup mandate flow and also in setupintent sync response.